src/pages/home/report/ReportActionItemSingle.js

```diff
@@ -114,14 +114,16 @@ function ReportActionItemSingle(props) {
     let secondaryAvatar = {};
     const primaryDisplayName = displayName;
     if (displayAllActors) {
-        const secondaryUserDetails = props.personalDetailsList[props.iouReport.ownerAccountID] || {};
+        // When we merge IOUs into one, if ownerAccountID is also actorAccountID, we should use managerID instead to prevent duplicate information.
```

Comment:
```suggestion
        // The ownerAccountID and actorAccountID can be the same if the a user requests money back from the IOU's original creator, in that case we need to use managerID to avoid displaying the same user twice
```